models/salesforce__opportunity_line_item_enhanced.sql

```diff
@@ -57,14 +57,10 @@ final as (
         {% endif %}
 
         --The below script allows for pass through columns.
-        {% if var('opportunity_line_item_pass_through_columns',[]) != [] %}
-        , oli.{{ var('opportunity_line_item_pass_through_columns') | join (", oli.")}}
-
-        {% endif %}
-
-        {% if var('product_2_pass_through_columns',[]) != [] %}
-        , product_2.{{ var('product_2_pass_through_columns') | join (", product_2.")}}
+        {{ fivetran_utils.persist_pass_through_columns(pass_through_variable='salesforce__opportunity_line_item_pass_through_columns', identifier='opportunity_line_item') }}
```

Comment:
When I try to pass a variable through here I get the following error.
![image](https://user-images.githubusercontent.com/74217849/236038003-6f16e96e-6fd3-4159-978c-c28e876f0cd3.png)

This needs to refer to the identifier of the table below. Since it is being aliased we need to properly add the alias here for the identifier.